charts/stable/jellyfin/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: jellyfin/jellyfin
   # -- image tag
-  tag: 10.7.7
+  tag: 10.8.1
```

Comment:
```suggestion
  # @default -- chart.appVersion
  tag:
```
By doing this we will only have to change the `appVersion` field in the Chart.yaml file in the future.

Reply:
Done - TIL that's a thing, cool feature

Reply:
Should the values table in `README.md` get an update as well?

Reply:
Normally that will be taken care of automatically when the chart gets merged :)

Reply:
Cool - in that case, this should be good to go